RELEASE/scripts/sl_ascend/sl_util.ash

```diff
@@ -3981,17 +3981,37 @@ boolean useCocoon()
 	skill cocoon = $skill[none];
 	if(have_skill($skill[Cannelloni Cocoon]))
 	{
+		skill blood_skill = $skill[none];
+		if(sl_have_skill($skill[Blood Bubble]) && sl_have_skill($skill[Blood Bond]))
```

Comment:
Can you also only maintain blood bond if familiars are available? Don't want to waste hp for nothing in pokefam, for example. I think there's at least one other path with standard classes/access to perms and no familiars too, isn't there?